src/test/java/thredds/server/metadata/util/NCMLModiferTest.java

```diff
@@ -0,0 +1,18 @@
+package thredds.server.metadata.util;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NCMLModiferTest {
+
+    @Test
+    public void testFormat() throws Exception {
+        NCMLModifier ncmlModifier = new NCMLModifier();
+        double dbl = -1.6391277313232422E-6;
+        String fmtDbl = ncmlModifier.format(dbl);
+
+        assertEquals("-.0", fmtDbl);
```

Comment:
Is this the desired result? To round `-1.6391277313232422E-6` to `-.0`? If you do something like:
```java
DecimalFormat dFmt = new DecimalFormat("0");
dFmt.setMaximumFractionDigits(340);
```
The value will be `-0.0000016391277313232422`. Never rounded and never in scientific notation. See http://stackoverflow.com/a/25307973/3874643